ENTITY/ClearEntityLastDamageEntity.md

```diff
@@ -8,7 +8,9 @@ ns: ENTITY
 cs_type(Any) void CLEAR_ENTITY_LAST_DAMAGE_ENTITY(Entity entity);
 ```
 
+This native **could effect** the arguments of the `CEventNetworkEntityDamage` game event, by clearing the damaging entity before the event is fired.
```

Comment:
`to effect` is not a verb, did you mean `affect`?

Reply:
Oops, yes.